packages/venia-ui/lib/components/ShoppingBag/shoppingBag.js

```diff
@@ -5,8 +5,16 @@ import { useScrollLock } from '@magento/peregrine';
 import { useShoppingBag } from '@magento/peregrine/lib/talons/ShoppingBag/useShoppingBag';
 import { mergeClasses } from '@magento/venia-ui/lib/classify';
 
+import ProductListing from './ProductListing';
+
+import ShoppingBadOperations from './shoppingBag.gql';
+
 import defaultClasses from './shoppingBag.css';
 
+const Error = () => {
+    return <div>TBD</div>;
```

Comment:
Do we wanna have an error message component/text in case of any errors while performing the remove mutation? Ill delete it if not needed.